hypertrace-metrics-generator/hypertrace-metrics-generator-api/build.gradle.kts

```diff
@@ -22,14 +22,9 @@ sourceSets {
 
 dependencies {
   implementation("com.google.protobuf:protobuf-java:3.23.3")
-  implementation("org.apache.kafka:kafka-clients:7.4.0-ccs")
+  implementation(platform("org.hypertrace.core.kafkastreams.framework:kafka-bom:0.2.14"))
```

Comment:
Here too, although this looks like somehow the api already depended on kafka clients so I guess not a regression

Reply:
kept as is, as it is using kafka-client.